bauhaus/workflows/secondary/datasetOps.py

```diff
@@ -13,8 +14,11 @@ def movieName(filename):
 
     base = op.basename(filename)
     fields = base.split(".")
-    assert len(fields) in (3, 4) and fields[-1] == "xml"
-    movieName = fields[0]
+    assert len(fields) > 2 and fields[-1] == "xml"
+    if len(fields) > 3 and re.match(r"chunk[0-9]+", fields[-3]):
+        movieName = ".".join(fields[:-3])
+    else:
+        movieName = ".".join(fields[:-2])
```

Comment:
With this change the method comment is no longer quite accurate.  The comment could be amended by removing/adjusting the first sentence; the *example* is still accurate.

On balance, I'm not sure about merging this change.  Technically, PacBio movie names do not contain "." characters, and I could argue that setting up filenames in this manner is a violation of bauhaus's implicit assumptions.  The solution, instead, would be to make such implicit assumptions *explicit*, rejecting such data at the `validate` step.